pkg/controller/logstorage/users/user_controller.go

```diff
@@ -159,7 +164,7 @@ func (r *UserController) Reconcile(ctx context.Context, request reconcile.Reques
 	}
 	if elasticsearch == nil || elasticsearch.Status.Phase != esv1.ElasticsearchReadyPhase {
 		r.status.SetDegraded(operatorv1.ResourceNotReady, "Waiting for Elasticsearch cluster to be operational", nil, reqLogger)
-		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
```

Comment:
Why can we get rid of this? I don't see a watch for the resource that `utils.GetElasticsearch` fetches and we don't return an error here so there won't be any requeue of this Reconcile.

Reply:
mmm I think this one was accidental. Good catch.

Reply:
Added a watch for Elasticsearches